ci/infra/openstack/worker-instance.tf

```
@@ -76,10 +76,10 @@ resource "openstack_compute_instance_v2" "worker" {
     name = "${var.internal_net}"
   }
 
-  security_groups = [
+  security_groups = "${var.enable_openstack_port_security == true ? [
```

I think this is missing the closing `}`.
```suggestion
  security_groups = "${var.enable_openstack_port_security} == true ? [
```